tests/integration/ambient/main_test.go

```diff
@@ -91,9 +91,6 @@ func TestMain(m *testing.M) {
 			cfg.DeployEastWestGW = false
 			cfg.ControlPlaneValues = `
 values:
-  pilot:
-    env:
-      ISTIO_MULTIROOT_MESH: true
```

Comment:
this flag was no longer needed after the `TestIntermediateCertificateRefresh` test was no longer modifying the root cert